### ISSUE TITLE
html code fails

### ISSUE THREAD

**Issue by dniemeier** [gdemin/expss#6]
This is really nice package. Thank you. I've run into an error that gets triggered (I think) when the table has a variable with categories that use a `$`. 

The code works fine within a md file, 
```
library(expss)
tempDF = apply_labels(workingDF,
                      date="Date",
                      ID="ID",
                      functions="Function"
)
cro_cases(tempDF$functions[tempDF$date==19],tempDF$ID[tempDF$date==19])
```

![image](https://user-images.githubusercontent.com/13838793/38761358-d0ab14e2-3f36-11e8-8215-84af931109d0.png)

but when I knit it, 

![image](https://user-images.githubusercontent.com/13838793/38761344-ba666f6a-3f36-11e8-9ec5-de7fc2e6af8f.png)

**Comment by gdemin** [comment]
Possibly it is a bug with htmlTable package which I use for knitting. I open an issue for it: https://github.com/gforge/htmlTable/issues/57

**Comment by gdemin** [comment]
This issue will be fixed with next release of expss/htmlTable. By now you can try as a quick workaround the following code:
```
tbl = cro_cases(tempDF$functions[tempDF$date==19],tempDF$ID[tempDF$date==19])
tbl[[1]] = gsub("\\$", "&#36;", tbl[[1]])
tbl
```

**Comment by dniemeier** [comment]
Thx. I think I’ll wait for fix. Using reg expressions to replace stuff
isn’t really a fix and to have to do this for every table I have with chars
that don’t play nice means the function really isn’t ready. I was testing
it for student use but I think there might be other options

Thanks.

**Comment by gdemin** [comment]
Fixed in v0.8.7
